packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -118,6 +133,15 @@ export class SDKClient {
     this.cacheService = cacheService;
     this.maxChunks = Number(process.env.FILE_APPEND_MAX_CHUNKS) || 20;
     this.fileAppendChunkSize = Number(process.env.FILE_APPEND_CHUNK_SIZE) || 5120;
+
+    this.mirrorNodeClient = new MirrorNodeClient(
+      process.env.MIRROR_NODE_URL || '',
+      logger.child({ name: `mirror-node` }),
+      register,
+      this.cacheService,
+      undefined,
+      process.env.MIRROR_NODE_URL_WEB3 || process.env.MIRROR_NODE_URL || '',
+    );
```

Comment:
Please, let's inject this `MirrorNodeClient` instead of instantiating it here, I see no point in having multiple copies of this client in the heap space. Same for other places where `MirrorNodeClient` is used, we should inject the same object instance for those classes. 

In general, it's a good practice to apply dependency injection to make each class independent of its dependencies and have a loosely coupled program.